README.md

```diff
@@ -51,7 +51,8 @@ Conversion-only quickstart (✅ Core):
 from megatron.bridge import AutoBridge
 
 # 1) Create a bridge from a Hugging Face model (hub or local path)
-bridge = AutoBridge.from_hf_pretrained("meta-llama/Llama-3.2-1B", trust_remote_code=True)
+# Set trust_remote_code to True only if you trust the repository
+bridge = AutoBridge.from_hf_pretrained("meta-llama/Llama-3.2-1B", trust_remote_code=trust_remote_code)
```

Comment:
for examples, isn't it fine if the user explicitly opts into this? why do we need this here? trust_remote_code isn't defined anywhere